lib/resque/worker.rb

```diff
@@ -200,9 +200,15 @@ def queues
     end
 
     def glob_match(pattern)
-      Resque.queues.select do |queue|
+      matches = Resque.queues.select do |queue|
         File.fnmatch?(pattern, queue)
-      end.sort
+      end
+
+      if ENV['SHUFFLE'] && ENV['SHUFFLE'] != 'false'
```

Comment:
Set up wildcard glob sort/shuffle behavior in `#initialize`

Reply:
What do think of making this a little more forgiving?

```suggestion
      if ENV['SHUFFLE'] && ENV['SHUFFLE'].downcase != 'false'
```